pkg/cliplugins/crd/plugin/options.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2022 The KCP Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package plugin
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"k8s.io/apimachinery/pkg/util/errors"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+
+	pluginbase "github.com/kcp-dev/kcp/pkg/cliplugins/base"
+)
+
+// Options for the crd commands.
+type Options struct {
+	*pluginbase.Options
+
+	Filename string
+	Prefix   string
+}
+
+// NewOptions provides an instance of Options with default values
+func NewOptions(streams genericclioptions.IOStreams) *Options {
+	return &Options{
+		Options: pluginbase.NewOptions(streams),
+	}
+}
+
+// BindFlags binds the arguments common to all sub-commands,
+// to the corresponding main command flags
+func (o *Options) BindFlags(cmd *cobra.Command) {
+	o.Options.BindFlags(cmd)
+
+	cmd.Flags().StringVarP(&o.Filename, "filename", "f", o.Filename, "Path to a file containing the CRD to convert to an APIResourceSchema, or - for stdin")
+	cmd.Flags().StringVar(&o.Prefix, "prefix", o.Prefix, "Prefix to use for the APIResourceSchema's name, before <resource>.<group>")
+}
+
+func (o *Options) Validate() error {
+	var errs []error
+
+	if err := o.Options.Validate(); err != nil {
+		errs = append(errs, err)
+	}
+
+	if o.Filename == "" {
+		errs = append(errs, fmt.Errorf("--filename is required"))
+	}
+
+	if o.Prefix == "" {
+		errs = append(errs, fmt.Errorf("--prefix is required"))
```

Comment:
Coming at this with 0 background, I don't know why this would be required?

Reply:
Added to example text.